src/main/steps/common/form/fields/inset.njk

```diff
@@ -0,0 +1,5 @@
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
```

Comment:
Where this common defined insert.njk is being used ?
Can we use if feasible else let's remove this please.